### PR TITLE
Add kserve install instrunctions

### LINKED ISSUE #2178: Kserve is not installed with Kubeflow 1.5
There is a problem when deploying kserve by installing all the components of kubeflow via `kustomize build ... kubectl apply -f -`, the installation it is not installing KServe but the previous version of KFServing, which messes up with the cluster roles for new profiles using the models after deploying them as `InferenceService`.

### PR DESCRIPTION
**Pull request by kairos03**
**Which issue is resolved by this Pull Request:**
Resolves #2178

**Description of your changes:**
Add Kserve install instructions for integrate with kubeflow without KFServing.

**Checklist:**
- [x] Unit tests pass:
  **Make sure you have installed kustomize == 3.2.1**
    1. `make generate-changed-only`
    2. `make test`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -272,6 +272,27 @@ the following command, but we recommend migrating to KServe as soon as possible:
 kustomize build apps/kfserving/upstream/overlays/kubeflow | kubectl apply -f -
 ```
 
+If you don't need KFServing and want to use kserve as integrated mode not stand-alone modee, edit kustomization.yaml of kserve.
```
Comment (rachitchauhan43) [inline README.md:275]:
```suggestion
If you don't need KFServing and want to use kserve in an integrated mode and not in standalone mode, then edit the kustomization.yaml of kserve.
```

**Review by rachitchauhan43 — COMMENTED** [review]
This will be really helpful for the community. Having just this comment in the yaml file, doesn't help much :
```
# For KF 1.5 we are including both KFServing and KServe. Thus we install the
# standalone kserve manifests, to avoid conflicts with 0.6.1 KFServing.
```

Thanks @kairos03  for contributing this.

**Comment by kairos03** [comment]
> This will be really helpful for the community. Having just this comment in the yaml file, doesn't help much :
> 
> ```
> # For KF 1.5 we are including both KFServing and KServe. Thus we install the
> # standalone kserve manifests, to avoid conflicts with 0.6.1 KFServing.
> ```
> 
> Thanks @kairos03 for contributing this.

Thank you for your comment. I reflect your commit. :)

**Comment by rachitchauhan43** [comment]
> > This will be really helpful for the community. Having just this comment in the yaml file, doesn't help much :
> > ```
> > # For KF 1.5 we are including both KFServing and KServe. Thus we install the
> > # standalone kserve manifests, to avoid conflicts with 0.6.1 KFServing.
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Thanks @kairos03 for contributing this.
> 
> Thank you for your comment. I reflect your commit. :) and plz agree CLA. @rachitchauhan43

Agreed to CLA.

**Comment by juliusvonkohout** [comment]
Please put this on the agenda of the next manifest wg meeting in two weeks and join us for a discussion

**Pull request closed without merging**